app/routes/schemas/agreementNumber.js

```diff
@@ -0,0 +1,10 @@
+const Joi = require('joi')
```

Comment:
Files are named in kebab-case rather than camelCase

Reply:
changed